engine/build.gradle

```
@@ -33,52 +33,58 @@ buildscript {
-    compile "com.github.marschall:zipfilesystem-standalone:1.0.1"
-    compile 'dom4j:dom4j:1.6.1'
+    implementation "com.github.zafarkhaja:java-semver:0.9.0" // gestalt lost this...
+    api "com.github.everit-org.json-schema:org.everit.json.schema:1.11.1"
+    implementation "com.github.marschall:zipfilesystem-standalone:1.0.1"
+    implementation 'dom4j:dom4j:1.6.1'
 
     //TODO inserted by someone who has no idea how the gradle thing works. Inspect whether necessary (for tests). Copied rom desktop build.gradle. May break things with android (maybe)
-    testCompile "com.badlogicgames.gdx:gdx-backend-headless:$gdxVersion"
-    testCompile "com.badlogicgames.gdx:gdx-platform:$gdxVersion:natives-desktop"
-    testCompile "com.badlogicgames.gdx:gdx-box2d-platform:$gdxVersion:natives-desktop"
-    testCompile "com.badlogicgames.gdx:gdx-controllers-desktop:$gdxVersion"
-    testCompile "com.badlogicgames.gdx:gdx-controllers-platform:$gdxVersion:natives-desktop"
+    testImplementation "com.badlogicgames.gdx:gdx-backend-headless:$gdxVersion"
+    testImplementation "com.badlogicgames.gdx:gdx-platform:$gdxVersion:natives-desktop"
+    testImplementation "com.badlogicgames.gdx:gdx-box2d-platform:$gdxVersion:natives-desktop"
+    testImplementation "com.badlogicgames.gdx:gdx-controllers-desktop:$gdxVersion"
+    testImplementation "com.badlogicgames.gdx:gdx-controllers-platform:$gdxVersion:natives-desktop"
 
     // Test lib dependencies
-    testCompile group: 'junit', name: 'junit', version: '4.12'
-    testCompile group: 'org.mockito', name: 'mockito-core', version: '2.7.22'
-    testCompile group: 'org.jboss.shrinkwrap', name: 'shrinkwrap-depchain-java7', version: '1.1.3'
-    testCompile group: 'org.assertj', name: 'assertj-core', version: '3.9.0'
+    testImplementation group: 'junit', name: 'junit', version: '4.12'
+    testImplementation group: 'org.mockito', name: 'mockito-core', version: '2.7.22'
+    testImplementation group: 'org.jboss.shrinkwrap', name: 'shrinkwrap-depchain-java7', version: '1.1.3'
+    testImplementation group: 'org.assertj', name: 'assertj-core', version: '3.9.0'
 }
 
 task cacheReflections {
     description = 'Caches reflection output to make regular startup faster. May go stale and need cleanup at times.'
-    // TODO: The extra "org" qualifier excludes test classes otherwise sucked up in Jenkins, causing issues. Redo later
-    File dirToReflect = new File(sourceSets.main.output.classesDir, "org")
-    inputs.files dirToReflect
-    outputs.file file(sourceSets.main.output.classesDir.toString() + "/reflections.cache")
+    inputs.files sourceSets.main.output.classesDirs,
+            // getClassesDir from all sourceSets (for any jvm (seems) language)
+            configurations."${sourceSets.main.runtimeClasspathConfigurationName}"
+
+    outputs.upToDateWhen {classes.state.upToDate}
+    outputs.file("$buildDir/resources/main/reflections.cache")
```

Where is `$buildDir/resources/main/reflections.cache` created? It seems that the reflections are saved in a different location. Did you mean to use `$buildDir/classes/reflections.cache` instead?
```suggestion
    outputs.file("$buildDir/classes/reflections.cache")
```